cobald_tests/daemon/core/test_config.py

```diff
@@ -30,6 +30,27 @@ def test_load(self):
                 assert True
             assert True
 
+    def test_load_invalid(self):
+        """Load a invalid YAML config (dangling parameter)"""
+        with NamedTemporaryFile(suffix=".yaml") as config:
+            with open(config.name, "w") as write_stream:
+                write_stream.write(
+                    """
+                    pipeline:
+                        - !LinearController
+                          low_utilisation: 0.9
+                          foo: 0
+                        - !MockPool
+                    """
+                )
+            try:
+                with load(config.name):
+                    assert False
+            except TypeError:
```

Comment:
Configuration errors like this throw multiple exceptions, one of these is a ConfigurationError, but the first one is a TypeError. Should I try to inspect and validate the arguments for the constructor or should I leave it this way? From users perspective it should not matter, since there is also a ConfigurationError.

Reply:
For the time being, it's fine if the error does not pass silently – there is no need to convert the ``TypeError`` into a ``ConfigurationError``.